modules/azure/src/main/java/org/testcontainers/azure/AzureEventhubsEmulatorContainer.java

```diff
@@ -0,0 +1,106 @@
+package org.testcontainers.azure;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+/**
+ * Testcontainers implementation for Azure Eventhubs Emulator.
+ * <p>
+ * Supported image: {@code "mcr.microsoft.com/azure-messaging/eventhubs-emulator"}
+ * <p>
+ * Exposed ports:
+ * <ul>
+ *     <li>AMQP: 5672</li>
+ *     <li>Kafka: 9092</li>
+ * </ul>
+ */
+public class AzureEventhubsEmulatorContainer extends GenericContainer<AzureEventhubsEmulatorContainer> {
+
+    private static final int DEFAULT_AMQP_PORT = 5672;
+
+    private static final int DEFAULT_KAFKA_PORT = 9092;
+
+    private static final String CONNECTION_STRING_FORMAT =
+        "Endpoint=sb://%s:%d;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=SAS_KEY_VALUE;UseDevelopmentEmulator=true;";
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse(
+        "mcr.microsoft.com/azure-messaging/eventhubs-emulator"
+    );
+
+    private AzuriteContainer azuriteContainer;
+
+    private MountableFile config;
+
+    /**
+     * @param dockerImageName specified docker image name to run
+     */
+    public AzureEventhubsEmulatorContainer(final DockerImageName dockerImageName) {
+        super(dockerImageName);
+        dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
+
+        waitingFor(Wait.forLogMessage(".*Emulator Service is Successfully Up!.*", 1));
+        withExposedPorts(DEFAULT_AMQP_PORT, DEFAULT_KAFKA_PORT);
+    }
+
+    @Override
+    public void start() {
+        if (this.azuriteContainer == null) {
+            this.azuriteContainer =
+                new AzuriteContainer(AzuriteContainer.DEFAULT_IMAGE_NAME.withTag("3.33.0")).withNetwork(getNetwork());
+        }
+        this.azuriteContainer.start();
+
+        super.start();
+    }
+
+    @Override
+    public void stop() {
+        super.stop();
+        if (this.azuriteContainer != null) {
+            this.azuriteContainer.stop();
+        }
+    }
+
+    /**
+     * Provide the broker configuration to the container.
+     *
+     * @param config The file containing the broker configuration
+     * @return this
+     */
+    public AzureEventhubsEmulatorContainer withConfig(final MountableFile config) {
+        this.config = config;
+        return this;
+    }
+
+    /**
+     * Accepts the EULA of the container.
+     *
+     * @return this
+     */
+    public AzureEventhubsEmulatorContainer acceptEula() {
+        return withEnv("ACCEPT_EULA", "Y");
```

Comment:
follow [this pattern](https://github.com/testcontainers/testcontainers-java/blob/daf44c454ef6b0507b2138c1bbfcab83cb35c11e/modules/mssqlserver/src/main/java/org/testcontainers/containers/MSSQLServerContainer.java#L76-L79) to check if license was accepted